docs/topics/mpp/mpp-connect-to-apis.md

```diff
@@ -72,7 +72,23 @@ internal actual fun writeLogMessage(message: String, logLevel: LogLevel) {
 The main rules regarding expected and actual declarations are:
 * An expected declaration is marked with the `expect` keyword; the actual declaration is marked with the `actual` keyword.
 * `expect` and `actual` declarations have the same name and are located in the same package (have the same fully qualified name).
-* `expect` declarations never contain any implementation code.
+* `expect` declarations never contain any implementation code and are abstract by default.
+
+> In interfaces, functions in actual declarations can be non-abstract and have a
+> body. To indicate that in `expect` declarations, explicitly mark functions as `open`:
+>
+> ```kotlin
+> expect interface Mascot {
+>     open fun name()
+> }
+>
+> actual interface Mascot {
```

Comment:
It would be useful to indicate somehow where the common code ends and where the platform code begins. For example, [this](https://kotlinlang.org/docs/kmm-connect-to-platform-specific-apis.html#example-send-and-receive-messages-from-a-websocket) article just uses separate code blocks with code-comments, like `// Common code` and `// Platform code`

I don't know whether we have a better way of representing this